### PR TITLE
Update how-to-use-http-detailed-errors-in-iis.md

### PR DESCRIPTION
**Pull request by ramakoni1**
Adding additional link that can be helpful for some customers.

### CODE REVIEW

**Comment by lextm** [comment]
It is rather strange to recommend an IIS 6 version article under an IIS 7 and above article. The description refer to so many old user interface elements that can easily confuse end users.

Even if you believe some ideas from that article is useful, you should try to write a new article that matches IIS 7 and above and then add a reference.

**Review by Rick-Anderson — APPROVED** [review]

File: iis/troubleshoot/diagnosing-http-errors/how-to-use-http-detailed-errors-in-iis.md [hunk 1]
```diff
@@ -178,3 +178,6 @@ Second, go to the `<httpErrors>` section in applicationHost.config and change it
 ## Summary
 
 Custom and Detailed Errors are powerful IIS features. They help you to troubleshoot problems without compromising the security of your IIS Server. Many configuration options help you to custom tailor your users' experience. Most importantly: experimenting with it is fun.
+
+## See Also
```
Comment (Rick-Anderson) [inline iis/troubleshoot/diagnosing-http-errors/how-to-use-http-detailed-errors-in-iis.md:182]:
```suggestion
## See also
```

File: iis/troubleshoot/diagnosing-http-errors/how-to-use-http-detailed-errors-in-iis.md [hunk 2]
```diff
@@ -178,3 +178,6 @@ Second, go to the `<httpErrors>` section in applicationHost.config and change it
 ## Summary
 
 Custom and Detailed Errors are powerful IIS features. They help you to troubleshoot problems without compromising the security of your IIS Server. Many configuration options help you to custom tailor your users' experience. Most importantly: experimenting with it is fun.
+
+## See also
+- [Troubleshooting Common IIS Errors](https://docs.microsoft.com/previous-versions/iis/6.0-sdk/ms524996(v=vs.90))
```
Comment (shirhatti) [inline iis/troubleshoot/diagnosing-http-errors/how-to-use-http-detailed-errors-in-iis.md:183]:
```suggestion
## See also

- [Troubleshooting Common IIS Errors](https://docs.microsoft.com/previous-versions/iis/6.0-sdk/ms524996(v=vs.90))
```

Comment (shirhatti) [reply iis/troubleshoot/diagnosing-http-errors/how-to-use-http-detailed-errors-in-iis.md:183]: Fix linter error

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**